src/_includes/contact.html

```diff
@@ -0,0 +1,68 @@
+<section id="reachout" class="row justify-content-center">
+  <div class="col-12 col-md-8 col-lg-6">
+    <h2 class="text-center text-md-start d-block mb-3">The time is now—reach out to help and to learn more</h2>
```

Comment:
This copy changed slightly (and is green in Ditto):

> The time is now—reach out to learn more

Reply:
Oh yea! Wow. That current copy kinda doesn't even make sense, `reach out to help`?! Will fix.

Reply:
updated: https://github.com/cal-itp/calitp.org/pull/357/commits/b4694899c42214bb944c12c8843058be0308f4f4